Makefile

```diff
@@ -111,7 +115,19 @@ fmt-dependencies:
 fmt: fmt-dependencies
 	find . -not \( -path "./.go" -prune \) -name "*.go" | xargs gofmt -s -w
 	find . -not \( -path "./.go" -prune \) -name "*.go" | xargs gofumpt -l -w
-	find . -not \( -path "./.go" -prune \) -name "*.go" | xargs gci -w -local "$(shell cat go.mod | head -1 | cut -d " " -f 2)"
+
+############################################################
+# lint section
+############################################################
+
+lint-dependencies:
```

Comment:
Can you make this the same as the other controllers?

For example:
https://github.com/stolostron/config-policy-controller/blob/20b3797521fe85e01b502430b5925fcac07b452d/Makefile#L90-L92